### DISCUSSION TITLE
How to pretty print a lua table. (Just like how to print `wezterm` table in `ShowDebugOverlay`)

### DISCUSSION THREAD

**Discussion by younger-1** [wezterm/wezterm#3586] [General]
I want to print the log by using `wezterm.log_info` which can't print table. I look through the util fuctions provided by wezterm and still can not get a way to pretty print the lua table. But I know could it could in the repl of `ShowDebugOverlay`, like the following:

![图片](https://user-images.githubusercontent.com/45989017/233762047-3ffcbab2-060b-44c1-9ec5-aec7cdcb6cd6.png)

**Comment by younger-1** [comment]
I know neovim use http://github.com/kikito/inspect.lua

**Comment by wez** [comment]
You can definitely log tables with wezterm.log_info:

```
> wezterm.log_info(os)
23:03:05.272 INFO logging > lua: {
    "clock": function,
    "date": function,
    "difftime": function,
    "execute": function,
    "exit": function,
    "getenv": function,
    "remove": function,
    "rename": function,
    "setlocale": function,
    "time": function,
    "tmpname": function,
}
```

**Reply by younger-1** [reply]
Wow, never notice that. I should have tried it myself. I neglected the docstring in the bottom ;)

![图片](https://user-images.githubusercontent.com/45989017/233768272-1e43831a-b70a-462a-b057-e7220a246086.png)
